pkgs/tools/virtualization/awsebcli/default.nix

```diff
@@ -0,0 +1,89 @@
+ { stdenv, python }:
+let
+  inherit (localPythonPackageSet) pkgs;
+  inherit (pkgs) fetchPypi buildPythonApplication;
+
+  localPythonPackageSet = python.override {
+    packageOverrides = self: super: rec {
+      docker = super.docker.overridePythonAttrs (oldAttrs: rec {
+          pname = "docker-py";
+          version = "1.7.2";
```

Comment:
note that you will have to update `name` as well yourself

Reply:
Thanks!